app/src/main/java/com/mapbox/maps/testapp/examples/SimpleMapActivity.kt

```diff
@@ -23,26 +23,11 @@ class SimpleMapActivity : AppCompatActivity() {
       }
   }
 
-  override fun onStart() {
-    super.onStart()
-    mapView.onStart()
-  }
-
-  override fun onStop() {
-    super.onStop()
-    mapView.onStop()
-  }
-
   override fun onLowMemory() {
```

Comment:
can we handle the `onLowMemory` callback as well in the lifecycle plugin? we could maybe use `Context. registerComponentCallbacks` API to observe the onLowMemory event.